src/system.h

```diff
@@ -90,7 +90,7 @@
 #    define SUPPORT_KEYBOARD
 #  endif
 
-#  ifdef USE_SDL_MIXER
+#  ifdef HAVE_SDL_MIXER
 #    define SUPPORT_WAV
```

Comment:
These defines are not used anyway, guess they were meant for the filefinder at some point.

Reply:
yeah, most of the defines are useless